src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -166,6 +169,16 @@ public HearingResponse sendCancelHearingRequest(HearingWrapper wrapper) {
         );
     }
 
+    private void sendPartiesNotifiedUpdateRequest(HearingWrapper wrapper) {
+        hmcHearingPartiesNotifiedApi.updatePartiesNotifiedHearingRequest(
```

Comment:
Don't we need to get a response here to make sure the request has gone through okay?